client/src/partials/cash/payments/templates/action.cell.html

```diff
@@ -23,5 +23,17 @@
         <span class="text-danger"><i class="fa fa-ban"></i> <span translate>FORM.LABELS.CANCELED</span></span>
       </a>
     </li>
+    <li class="divider"></li>
+    <li>
+      <a ui-sref="patientRegistry({ filters : { display_name : row.entity.patientName }})" data-method="invoiceReceipt" href>
```

Comment:
Should this data method be `invoiceReceipt` ? I can't tell exactly from the code but it looks like this opens a patient card, not an invoice.